pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -657,7 +658,7 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 			container: &pod.Spec.Containers[idx],
 			message:   message,
 		}
-		klog.V(2).Infof("Container %q (%q) of pod %s: %s", container.Name, containerStatus.ID, format.Pod(pod), message)
+		klog.V(2).InfoS("Message for Container of pod", "containerName", container.Name, "containerStatusID", containerStatus.ID, "pod", klog.KObj(pod), "msg", message)
```

Comment:
```suggestion
		klog.V(2).InfoS("Message for Container of pod", "containerName", container.Name, "containerStatusID", containerStatus.ID, "pod", klog.KObj(pod), "containerMessage", message)
```
"msg" is reserved for log message in JSON logs